src/robusta/integrations/zulip/sender.py

```
@@ -13,7 +13,7 @@
     TableBlock,
 )
 from robusta.core.reporting.base import BaseBlock, Finding, FindingStatus, LinkType
-from robusta.core.reporting.blocks import FileBlock, LinksBlock
+from robusta.core.reporting.blocks import FileBlock, LinksBlock, EmptyFileBlock
```

Ruff (0.12.2)

Redefinition of unused `FileBlock` from line 8

(F811)